modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+//* architecture/installation-.adoc
+
+[id="registry-configuring-storage-baremetal_{context}"]
+= Configuring registry storage for bare metal
+
+As a cluster administrator, following installation you must configure your
+registry to use storage.
+
+.Prerequisites
+
+* Cluster administrator permissions.
+* A cluster on bare metal.
+* PV with `ReadWriteMany` access mode, such as Azure File or NFS.
+* Must have "100Gi" capacity.
+
+.Procedure
+
+. To configure your registry to use storage, change the `spec.storage.pvc` in
```

Comment:
@adambkaplan @dmage - would either of you be able to provide the procedures for these two steps or point me to where I can find them? Thanks!

. To configure your registry to use storage, change the `spec.storage.pvc` in
the `configs.imageregistry/cluster` resource.

. Provide a suitable persistent volume.